### DISCUSSION TITLE
How to use mongoDb Collection::watch() in laravel

### DISCUSSION THREAD

**Discussion by robod9590** [jenssegers/laravel-mongodb#2652] [Q&A]
i want to use mongodb collection::watch()  in laravel so when change occure in mongodb collection i can update my mysql table
i connect both mysql and mongodb in laravel

**Comment by GromNaN** [comment]
Hello, the best solution is to use the `mongodb/mongodb` features directly. 

```php
/** @var \MongoDB\Laravel\Connection */
$mongodb = DB::connection('mongodb');
```
